doxia-core/src/main/java/org/apache/maven/doxia/sink/impl/Xhtml5BaseSink.java

```diff
@@ -1112,6 +1109,24 @@ && getCellJustif() != null) {
         }
     }
 
+    private static String getStyleForTableJustification(int justification) {
```

Comment:
The actual code isn't just necessarify for tables only, it could apply everywhere logically, no?

Reply:
The constant defining the justification are only referring to tables according to their comments. Also nowhere else there is a Sink API taking into for justification…

Reply:
Right